src/styles/input.css

```
@@ -1670,6 +1772,66 @@
+}
+
+/* Opacity blink for whole-diagram grayscale handshaking phase */
+@keyframes swap-route-blink {
+  0%, 100% { opacity: 0.55; }
+  50% { opacity: 1; }
+}
+
+/* Glow pulse for route segments (yellow during contracting, green during settling) */
+@keyframes swap-route-glow-blink {
+  0%, 100% {
+    filter: drop-shadow(0 0 4px currentColor);
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)